### PR TITLE
Bumping version to 5.8

### PR DESCRIPTION
**Pull request by maysamyabandeh**
@override-unit-failures

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Rocksdb Change Log
 ## Unreleased
+
+## 5.7.8 (08/14/2017)
```
Comment (ajkr) [inline HISTORY.md]: do you mean 5.8.0?

Comment (maysamyabandeh) [inline HISTORY.md]: :) yeah thanks for catching it.

File: HISTORY.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Rocksdb Change Log
 ## Unreleased
+
+## 5.8.0 (08/14/2017)
 ### New Features
 * Add Iterator::Refresh(), which allows users to update the iterator state so that they can avoid some initialization costs of recreating iterators.
 * Replace dynamic_cast<> (except unit test) so people can choose to build with RTTI off. With make, release mode is by default built with -fno-rtti and debug mode is built without it. Users can override it by setting USE_RTTI=0 or 1.
```
Comment (yiwu-arbug) [inline HISTORY.md]: I doubt whether "release mode is by default built with -fno-rtti and debug mode is built without it" is true. Looking at the Makefile there's no such logic.

Comment (siying) [inline HISTORY.md]: https://github.com/facebook/rocksdb/blob/master/Makefile#L101-L118

Comment (yiwu-arbug) [inline HISTORY.md]: Thanks @siying! Makefile don't have indentation and I miss line 101.

**Pull request closed without merging**